src/components/CollaboratorModal/components/MainSubmodal.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  ModalHeader,
+  ModalBody,
+  Grid,
+  GridItem,
+  Text,
+  Box,
+  Divider,
+  FormControl,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalProps,
+  useFormControlContext,
+  Skeleton,
+  Stack,
+} from "@chakra-ui/react"
+import {
+  IconButton,
+  ModalCloseButton,
+  FormErrorMessage,
+  FormLabel,
+  Button,
+} from "@opengovsg/design-system-react"
+import _ from "lodash"
+import { useEffect, useState } from "react"
+import { FormProvider, useForm } from "react-hook-form"
+import { BiTrash } from "react-icons/bi"
+import { useParams } from "react-router-dom"
+
+import { useLoginContext } from "contexts/LoginContext"
+
+import * as CollaboratorHooks from "hooks/collaboratorHooks"
+
+import { Collaborator } from "types/collaborators"
+import { MiddlewareError } from "types/error"
+import { DEFAULT_RETRY_MSG, useSuccessToast } from "utils"
+
+import { ACK_REQUIRED_ERROR_MESSAGE } from "../constants"
+
+import { AcknowledgementSubmodalContent } from "./AcknowledgementSubmodal"
+
+const LAST_LOGGED_IN_THRESHOLD_IN_DAYS = 60
+
+const numDaysAgo = (previousDateTime: string): number => {
+  const currDateTime = new Date(Date.now())
+  const prevDateTime = new Date(previousDateTime)
+  return Math.floor(
+    (currDateTime.getTime() - prevDateTime.getTime()) / (60 * 60 * 24 * 1000)
+  )
+}
+
+interface CollaboratorListProps {
+  onDelete: (user: Collaborator) => void
+}
+
+const CollaboratorListSection = ({ onDelete }: CollaboratorListProps) => {
+  const { email } = useLoginContext()
+  const { siteName } = useParams<{ siteName: string }>()
+  const {
+    data: collaborators,
+    isError,
+  } = CollaboratorHooks.useListCollaboratorsHook(siteName)
+  const { isDisabled } = useFormControlContext()
+
+  return (
+    <Box mt="40px">
+      {collaborators?.map((collaborator: Collaborator) => (
+        <>
+          <Grid
+            templateColumns="repeat(11, 1fr)"
+            h="56px"
+            key={`collaborator-${collaborator.id}`}
+          >
+            <GridItem colSpan={8}>
+              <Box display="flex" alignItems="center" h="100%">
+                <Text>{collaborator.email}</Text>
+                <Text pl="4px" color="gray.500">
+                  {email === collaborator.email ? "(You)" : null}
+                </Text>
+                {numDaysAgo(collaborator.lastLoggedIn) >=
+                  LAST_LOGGED_IN_THRESHOLD_IN_DAYS && (
+                  <Text pl="4px" color="danger.500">
+                    {`(Last logged in ${numDaysAgo(
+                      collaborator.lastLoggedIn
+                    )} days ago)`}
+                  </Text>
```

Comment:
I think the text here should use 0.875rem instead. It's currently too large and causing an overflow:

<img width="643" alt="Screenshot 2022-10-20 at 11 04 04" src="https://user-images.githubusercontent.com/27919917/196846657-8c3f46f8-2a43-4d07-a8d1-ec7b0a7b0898.png">

Also, minor nit: Can we call `numDaysAgo()` once and store the result in a variable, then just use that variable here?

Reply:
this text area is prone to overflowing - if we have an extremely long email, it will overflow its boundary too :/

more inclined to file as issue cos this is cosmetic and has no functional impact (atm) 

will fix the multiple calls

Reply:
Yeah I think we don't have to fix the overflowing issue, only the text size should still be 0.875rem as defined in Figma.